packages/components/Notification/src/NotificationTokens.ios.ts

```diff
@@ -3,6 +3,16 @@ import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { NotificationTokens } from './Notification.types';
 import { DynamicColorIOS } from 'react-native';
 
+const emptyShadowStyle = {
+  ambient: { x: 0, y: 0, blur: 0, color: '#00000000' },
+  key: { x: 0, y: 0, blur: 0, color: '#00000000' },
+};
+
+const notificationShadowStyle = {
+  ambient: { x: 0, y: 8, blur: 8, color: '#00000024' },
+  key: { x: 0, y: 0, blur: 1, color: '#0000001f' },
+};
+
```

Comment:
@TravisSpomer Not sure if you're the correct person to ask for this. Is there a way we can get this "shadow0" style into the token pipeline? Basically a token to say "render no shadows". I know it sounds a little counterintuitive, but would help simplify some code on our end :)

Reply:
@saadnajmi Yeah, I checked and there isn't an "official" `shadow0` token, but it seems entirely reasonable to me. If your non-iOS colleagues are fine with it we can add it to the shadow tokens, or it can just be part of the exported iOS code with no actual token backing it. Start up a group chat or just let me know which y'all prefer and I'll help out wherever I can.

Reply:
My only two cents here is that we should look into whether setting any shadow at all has any render implications. Like if cycles are spent drawing an invisible shadow/it ends up inserting some new element because there's a shadow. It's not great to end up with two extra Views in the UI tree if they don't add value.

Reply:
Agreed 100%. I hereby clarify my response to say that it seems entirely reasonable to me *only if it can be done in a way that does not impact performance.* 🙂 I'm not familiar enough (well, at all) with iOS dev to have any specific advice there.

Reply:
Agree that not rendering a shadow is the preferable option. The issue I saw we ran into is this:

Control A by default renders shadow8, so its at the top level of controlATokens.ts,

```
controlATokens: {
   color: t.colors.brandForeground
   shadow: t.shadows.shadow8
   ...
}
```


Control A's variant "foo" does not render a shadow ( in this case, Notification's variant "bar"). We generally override this by nesting some control token definitions inside the controlATokens.ts file. But, what do we override the shadow as?


```
controlATokens: {
   color: t.colors.neutralForeground
   shadow: t.shadows.shadow8
   ...
   foo: {
     color: t.colors.brandForeground
     shadow: ???
   }
}
```
We can pass `undefined`, and that could work, but it looked kinda hacky. We could alternatively in the `Shadow` component check if the shadow-to-render is shadow) or undefined, and then not render 2 extra views and just return `{children}`. Regardless, this is one of the questions that came up as we implemented shadows both as a slot and into some controls. We can follow up offline :)

Reply:
Note that this issue is specific to how FURN implements tokens and overriding per variant, @TravisSpomer .I guess we should decide what we want to do internally in FURN before making a design ask.

Reply:
Sure, no problem. Just send me a boop once you've settled on what would work best.